packages/Textarea/src/Textarea.js

```diff
@@ -85,7 +82,9 @@ function Textarea(props) {
 
   const setRef = node => {
```

Comment:
I don't think we need the setRef at all any more right? this only is required if we have to support react 15 that we don't use in our platform.

Reply:
I think we still need it here, because the main objective is to assign `textareaRef` and also pass `ref` to consumer side and the `textareaRef` has been used here https://github.com/acl-services/paprika/pull/599/files#diff-a642316bf41d6a04a807281a1985e762R46